### PR TITLE
Fixed: test intervals accepts negative values

### PR DESCRIPTION
**Pull request by yoayushraj99**
Issue: #455 

@aquibbaig Could you please review my changes?

### CODE REVIEW

File: local-config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ utils:
 test_interval:
 - test: ping
   type: sec
-  duration: 20
+  duration: 14
```
Comment (aquibbaig) [inline local-config.yml:14]: Let's drop this change as a part of this PR.

File: dashboard/v1.1/src/pages/Config/components/IntervalDetails.tsx [hunk 2]
```diff
@@ -3,12 +3,16 @@ import { TextField, Button, Typography } from '@material-ui/core';
...
+  const [inputValue, setInputValue] = useState<Number>(
     props.durationValue || ''
   );
 
   const handleIntervalOnChange = e => {
-    setInputValue(e.target.value);
+    if (e.target.value >= 0) {
+      setInputValue(e.target.value);
+    } else {
+      alert('Please try a positive int value');
```
Comment (aquibbaig) [inline dashboard/v1.1/src/pages/Config/components/IntervalDetails.tsx:14]: Can we have a general-purpose component to handle alerts? https://material-ui.com/components/alert/#alert

**Review by aquibbaig — CHANGES_REQUESTED** [review]
Thanks for handling this edge case. Minor fixes and it looks good to go

File: local-config.yml (hunk 1 above)
Comment (yoayushraj99) [reply local-config.yml]: ok i will drop this change

File: dashboard/v1.1/src/pages/Config/components/IntervalDetails.tsx (hunk 2 above)
Comment (yoayushraj99) [reply dashboard/v1.1/src/pages/Config/components/IntervalDetails.tsx]: ok i will do these changes.

**Comment by yoayushraj99** [comment]
hey @aquibbaig i closed this pull request and opened a new one at #472

**Pull request closed without merging**